root/release/edit/information.tt

```diff
@@ -328,6 +328,11 @@
     <p>
       [% l('The purpose of this field is to add information that usually doesn\'t fit into the strict structural data schema of MusicBrainz (be it due to technical limitations that may be addressed later, or because the information in itself has to be free-text).') %]
     </p>
+    <p>
+      [% l('This annotation will be published under an open license (<a href="{url}" target="_blank">CC BY-NC-SA 3.0 </a>) and as such, it should not contain text that you don't have the right to release under that license. 
```

Comment:
The string currently stops at `don`. Single quote `'` in `don't` should be escaped with backslash `\`.